content/tutorials/channel-rewind.textile

```diff
@@ -0,0 +1,217 @@
+
+Create a new HTML file and add the following to include the drop down for various city options and a box to display the information:
+
+```[html]
+<html>
+<head>
+    <title>Ably's rewind channel param tutorial</title>
+    <!-- CDN for JQuery -->
+    <script src="https://code.jquery.com/jquery-3.4.1.min.js"
+        integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
+    <!-- CDN for Ably JS library -->
+    <script src="https://cdn.ably.io/lib/ably.min-1.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div>
+    City:
+    <!-- Drop down to select from the available cities -->
+    <select id="city-id">
+        <option value="2643741">London, UK</option>
+        <option value="1850147">Tokyo, Japan</option>
+        <option value="1273294">Delhi, India</option>
+        <option value="1796236">Shanghai, China</option>
+        <option value="3448439">Sao Paulo, Brazil</option>
+        <option value="5128581">New York, US</option>
+        <option value="360630">Cairo, Egypt</option>
+        <option value="2988507">Paris, France</option>
+        <option value="2950158">Berlin, Germany</option>
+        <option value="756135">Warsaw, Poland</option>
+        <option value="2147714">Sydney, Australia</option>
+    </select>
+    </div>
+    <br/>
+    <table style="width:100%; padding: 20px; height: 50px;" border="1">
+    <tr>
+    <!-- Text Field to show the temperature info for the selected city -->
+    <td style="text-align: center;">
+        <div class="container">
+        <h4 id="result-open-weather"></h4>
+        </div>
+    </td>
+    </tr>
+    </table>
+</body>
+</html>
+```
+
+Next, we'll instantiate the Ably library and attach to the channels with the rewind param enabled. With Ably version lower than 1.2, the rewind param needs to be added as part of the channel qualifier within the square brackets. If the channel already has a qualifier like @[meta]*@ or like in our case @[product:ably-openweathermap/weather]*@, we can simply append the existing qualifier name with @?rewind=<position>@ or @?rewind=<time>@ 
+
+Add the following after the closing @<body>@ tag of the HTML file above to instantiate the Ably library and declare some variables which we'll use later:
+
+```[html]
+<script type="text/javascript">
+  window.addEventListener("load", function () {
+      var ably = new Ably.Realtime("<YOUR-API-KEY>"),
+          $resultWeather = $('#result-open-weather')
+          channelWeather;
+  )};
+</script>
+```
+
+Make sure to add in your own Ably API Key in the above code snippet.
+
+What we need now to get the data from the relevant channels is to retrieve the user selection from the drop down menu and use the correlated code to subscribe to that particular channel. We'll add the rewind functionality here and set value to 1, signifying the number of previously published messages we'd like to retrieve from this data stream, on attaching to a channel.
```

Comment:
I think could be simpler first line,  `We now need to retrieve the user selection...` or something.